worker/src/RTC/RtpStreamRecv.cpp

```diff
@@ -284,6 +284,10 @@ namespace RTC
 		// Increase media transmission counter.
 		this->mediaTransmissionCounter.Update(packet);
 
+		// Ignore padding-only packets and don't mark the stream as active with them.
+		if (packet->GetPayloadLength() == 0)
```

Comment:
This is missing in `RtpStreamRecv::ReceiveRtxPacket`, right?

Reply:
This is resolved, right?

Reply:
@ggarber is this resolved?